src/tests/secp256k1_blake160_multisig_all.rs

```diff
@@ -32,14 +33,16 @@ fn test_multisig_script_hash() {
     let keys = generate_keys(3);
     let multi_sign_script = gen_multi_sign_script(&keys, 2, 0);
     let args = blake160(&multi_sign_script);
+    let lock_script = gen_multi_sign_lock_script(args.clone());
```

Comment:
It is weird: this is only used in assertion. We use a function like below:
```Rust
pub fn assert_script_error(err: Error, err_code: i8) {
    let error_string = err.to_string();
    assert!(
        error_string.contains(format!("error code {}", err_code).as_str()),
        "error_string: {}, expected_error_code: {}",
        error_string,
        err_code
    );
}
```

Reply:
Yeah I was mainly on the smallest change to get the tests back working, while also testing full equalness for the error object. Personally I think either way works, I don't have a preference.